dockstore/dockstore-support
# SEAB-6360: download github event from s3

**Description**
This PR creates a module `githubdelivery` and adds a command `download-event` to return the github payload from s3 from its key.

**Review Instructions**

It was the easiest to run on CloudShell 
Steps I used was 
1. Install Java `sudo yum install java-17-amazon-corretto`
2. clone the repo
3. Build only the `githubdelivery` module by running `./mvnw install -pl githubdelivery -am`
4. Configure your `github-delivery.config` for QA (template in `templates` folder)
5. Go to the module and run `java -jar target/githubdelivery-1.16.0-SNAPSHOT.jar download-event -k "2024-05-15/004701ac-1307-11ef-94b7-6b9aec991740"` and verify the payload is printed on the console.

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-6360

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install` in the project that you have modified (until https://ucsc-cgl.atlassian.net/browse/SEAB-5300 adds multi-module support properly)
- [x] If you are changing dependencies, check with dependabot to ensure you are not introducing new high/critical vulnerabilities
